config/.isort.cfg

```diff
@@ -0,0 +1,11 @@
+[settings]
+line_length = 120
+not_skip = __init__.py
+skip = migrations,south_migrations
+multi_line_output = 4
+force_single_line = False
+balanced_wrapping = True
+default_section = THIRDPARTY
+forced_separate = test_appsettings
+known_first_party = appsettings
+sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
```

Comment:
* I believe the `sections` are set to default, so we can delete them.
 * `forced_separate = test_appsettings` seems to be unused.